openshift/helper/ansible.py

```diff
@@ -606,8 +606,6 @@ def add_meta(prop_name, prop_prefix, prop_alt_prefix):
             if prop_alt_prefix != prop_prefix:
                 if prop_alt_prefix:
                     args[prop_prefix + prop_name]['aliases'] = [prop_alt_prefix + prop_name]
-                elif prop_prefix:
```

Comment:
I think rather than removing these branches entirely, it might be better to just add a check that ensures it's not creating an alias that conflicts with a top level key.

Reply:
That may even just entail keeping a set of all names + aliases that have been used and checking that everything we're setting is unique.

Reply:
Yes, this make sense. Will consider this option too